src/_common/content/content-editor/content-editor.vue

```diff
@@ -4,13 +4,21 @@
 			class="content-container"
 			:class="{
 				disabled: disabled,
-				'content-container-with-gutter': couldShowEmojiPanel,
+				'content-container-with-gutter-1': editorGutterSize === 1,
+				'content-container-with-gutter-2': editorGutterSize === 2,
 			}"
 			:style="{
 				minHeight: containerMinHeight,
 			}"
 		>
-			<div class="-doc" :class="editorStyleClass" ref="doc" />
+			<app-scroll-scroller @scroll.native="onScroll">
```

Comment:
I set it to `thin`.
And regarding the "content area", I agree, but it's basically impossible the way this is set up: The emoji and gif buttons are technically outside the content area, in the margin of the doc element. Since the doc element is the one we scroll, the scrollbar will be to the left of the buttons.
This is set up this way so there will never be text or other content behind the gif/emoji buttons.